src/statics/About.js

```diff
@@ -11,42 +11,90 @@ export default () =>
     <div className="mt-5 text-center">
       <h1><FormattedMessage id="about" /></h1>
       <p className="container pb-5">
-        The Busy project’s mission is to connect everyone to the next-gen
-        decentralized social network that gives voice and value back to people,
-        by building together a true sharing economy, a more free and trustworthy
-        Internet without middleman.
+        <FormattedMessage
+          id="@statics/busy_mission"
+          defaultMessage="The Busy project’s mission is to connect everyone to the next-gen decentralized social network that gives voice and value back to people, by building together a true sharing economy, a more free and trustworthy Internet without middleman"
+        />
+        .
       </p>
       <div className="About__1 hero">
         <div className="container">
-          <h1>Ensuring compensation for the creators of value</h1>
-          <p>Artists and User-generated content create billions dollars worth of
-            value absorbed by the shareholders of these intermediaries and centralized
-            services. As a non-profit and decentralized organization, Busy provides
-            the power of cryptocurrency and blockchain innovations within a platform
-            that returns much of its value to the creators ensuring fair compensation.</p>
+          <h1>
+            <FormattedMessage
+              id="@statics/ensure_compensation"
+              defaultMessage="Ensuring compensation for the creators of value"
+            />
+          </h1>
+          <p>
+            <FormattedMessage
+              id="@statics/busy_mission_2"
+              defaultMessage="Artists and User-generated content create billions dollars worth of value absorbed by the shareholders of these intermediaries and centralized services"
+            />
+            .
+            {' '}
+            <FormattedMessage
+              id="@statics/busy_mission3"
+              defaultMessage="As a non-profit and decentralized organization, Busy provides the power of cryptocurrency and blockchain innovations within a platform that returns much of its value to the creators ensuring fair compensation"
+            />
+            .
```

Comment:
Here too, i think you have to include all of them into the defaultMessage. Thai language dont have dot for example.